### ISSUE TITLE
BankImport: local variable 'basestring' referenced before assignment

### ISSUE THREAD

**Issue by jHetzer** [libracore/erpnextswiss#64]
### Expected behaviour
Import bank csv export

### Actual behaviour
Parse throws: UnboundLocalError: local variable 'basestring' referenced before assignment
```
Traceback (most recent call last):
  File "/home/frappe/frappe-dev/apps/frappe/frappe/app.py", line 61, in application
    response = frappe.handler.handle()
  File "/home/frappe/frappe-dev/apps/frappe/frappe/handler.py", line 21, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-dev/apps/frappe/frappe/handler.py", line 56, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-dev/apps/frappe/frappe/__init__.py", line 1030, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-dev/apps/erpnextswiss/erpnextswiss/erpnextswiss/page/bankimport/bankimport.py", line 658, in parse_file
    new_records = parse_by_template(content, bank_doc[0]['csv_template'], account, auto_submit, debug)
  File "/home/frappe/frappe-dev/apps/erpnextswiss/erpnextswiss/erpnextswiss/page/bankimport/bankimport.py", line 766, in parse_by_template
    { 'field_index': getFieldDefinition('booked_at_field'),'field_reg': getFieldDefinition('booked_at_reg'),'match_group': 'booked','regired': True},
  File "/home/frappe/frappe-dev/apps/erpnextswiss/erpnextswiss/erpnextswiss/page/bankimport/bankimport.py", line 717, in getFieldDefinition
    if isinstance(value, basestring()):
UnboundLocalError: local variable 'basestring' referenced before assignment
```

### Observed in (version)
Installed Apps
ERPNext: v11.1.20 (master)
ERPNextSwiss: v1.4.0 (develop)
Frappe Framework: v11.1.21 (master)
Python Version: 2.7.5 (default, Apr 9 2019, 14:30:50) [GCC 4.8.5 20150623 (Red Hat 4.8.5-36)]
CentOS Version: CentOS Linux release 7.6.1810 (Core)

### Steps to reproduce
Install latest CentOS
Install Frappe / ERPNext (master)
Install ERPNextSwiss  (develop)
Try to import csv export

**Comment by jHetzer** [comment]
Can be fixed by using `six.string_types`
https://github.com/jHetzer/erpnextswiss/commit/5f52d239bfde61ae860718958eeac1aa6b5283f4

**Comment by lasalesi** [comment]
Thanks, this has also been reported from other users on Python 2.7.
The above fix was applied in d64596f
